app/models/asset.rb

```diff
@@ -137,10 +137,7 @@ def image?
   end
 
   def etag_from_file
-    "%<mtime>x-%<size>x" % {
-      mtime: last_modified_from_file,
-      size: file_stat.size,
-    }
+    format("%<mtime>x-%<size>x", mtime: last_modified_from_file, size: file_stat.size)
```

Comment:
Feels like string interpolation might be nicer here, but this is fine too.

Reply:
Yeah, I see what you mean. Apparently `x` means "format as hex", which is quite obscure; that said, the interpolation approach - `to_s(16)` is also obscure!